python/brainyflow.py

```
@@ -1,115 +1,129 @@
-# python/brainyflow.py
-
 import asyncio
 import copy
 import warnings
 from abc import ABC, abstractmethod
-from typing import Any, Dict, List, Optional, Tuple, TypeVar, Generic, Callable, Union, Type, cast
+from typing import Any, Dict, List, Optional, Tuple, TypeVar, Generic, Callable, Union, Type, cast, TypedDict, Literal, overload, Awaitable, Sequence
```

Ruff (0.8.2)

`typing.Type` imported but unused

Remove unused import: `typing.Type`

(F401)